app/src/lib/components/PullRequestCard.svelte

```diff
@@ -34,21 +35,27 @@
 	let mergeableState: string | undefined;
 	let checksStatus: ChecksStatus | null | undefined = undefined;
 	let lastDetailsFetch: Readable<string> | undefined;
-	let lastChecksFetch: Readable<string> | undefined;
 
 	$: pr$ = githubService.getPr$($branch.upstreamName);
 	$: if ($branch && $pr$) updateDetailsAndChecks();
 
-	$: checksIcon = getChecksIcon(checksStatus, isFetchingChecks);
-	$: checksColor = getChecksColor(checksStatus);
-	$: checksText = getChecksText(checksStatus);
-	$: statusIcon = getStatusIcon(detailedPr);
-	$: statusColor = getStatusColor(detailedPr);
-	$: statusLabel = getPrStatusLabel(detailedPr);
+	$: checksTagInfo = getChecksTagInfo(checksStatus, isFetchingChecks);
+	$: infoMessageInfo = getInfoMessageInfo(
```

Comment:
Maybe rename this just `infoMessage`? That way it gets used like this:

```svelte
{#if infoMessage}
  <... icon={infoMessage.icon} />
{/if}
```

Reply:
Thanks! Updated it to `infoMessage` but didn't use spread operator to add props, because the `InfoMessage` using `slot` inside and its color type slight different.